docs/DeveloperGuide.md

```diff
@@ -239,6 +239,33 @@ _{more aspects and alternatives to be added}_
 
 _{Explain here how the data archiving feature will be implemented}_
 
+### View Command Feature
+
+The view mechanism is facilitated by ``DoConnek Pro``
```

Comment:
I feel it doesn't make much sense to have `DoConnek Pro` here randomly, it might be better to introduce the names of the classes or UI elements involved in  the view mechanism if you'd like to include this sentence at all.